src/lib/sql/table_privileges.sql

```diff
@@ -0,0 +1,75 @@
+-- Despite the name `table_privileges`, this includes other kinds of relations:
+-- views, matviews, etc. "Relation privileges" just doesn't roll off the tongue.
+--
+-- For each relation, get its relacl in a jsonb format,
+-- e.g.
+--
+-- '{postgres=arwdDxt/postgres}'
+--
+-- becomes
+--
+-- [
+--   {
+--     "grantee": "postgres",
+--     "grantor": "postgres",
+--     "is_grantable": false,
+--     "privilege_type": "INSERT"
+--   },
+--   ...
+-- ]
+select
+  c.oid as relation_id,
+  nc.nspname as schema,
+  c.relname as name,
+  case
+    when c.relkind = 'r' then 'table'
+    when c.relkind = 'v' then 'view'
+    when c.relkind = 'm' then 'materialized view'
+    when c.relkind = 'f' then 'foreign table'
+    when c.relkind = 'p' then 'partitioned table'
```

Comment:
Do you think these should have underscores? E.g. `materialized_view`

Reply:
Can do